### PR TITLE
add annotation support for serviceaccounts in helm-chart

### PR DESCRIPTION
**Pull request by imdhruva**

### CODE REVIEW

File: reportportal/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "5.3.5"
 description: ReportPortal.io AI-powered Test Automation Dashboard
 name: reportportal
-version: 5.3.5
+version: 5.3.6
```
Comment (HardNorth) [inline reportportal/Chart.yaml:5]: Please update the PR.

**Review by HardNorth — CHANGES_REQUESTED** [review]

**Comment by kartik-moolya** [comment]
@imdhruva  just a question, have you tried this with IAM role for service Account? (Im assuming you need annotations to service account for that reason) If yes, could you please share a dummy `values.yaml` that you used

**Comment by HardNorth** [comment]
@imdhruva Ping

**Comment by HardNorth** [comment]
Merge in: https://github.com/reportportal/kubernetes/pull/187

**Pull request closed without merging**